influxdb3_write/src/write_buffer/persisted_files.rs

```diff
@@ -34,6 +35,12 @@ impl PersistedFiles {
 
     /// Get the list of files for a given database and table, always return in descending order of min_time
     pub fn get_files(&self, db_id: DbId, table_id: TableId) -> Vec<ParquetFile> {
+        let three_days_ago = SystemTime::now()
```

Comment:
We should also add a check on `inner.add_persisted_snapshot` that will remove any files that meet this criteria. No reason to keep that metadata in RAM if it's not going to be used and over time it could get big if they keep a server running for a long time.